doc/languages-frameworks/cuda.section.md

```diff
@@ -68,16 +68,45 @@ All new projects should use the CUDA redistributables available in [`cudaPackage
 ### Updating CUDA redistributables {#updating-cuda-redistributables}
 
 1. Go to NVIDIA's index of CUDA redistributables: <https://developer.download.nvidia.com/compute/cuda/redist/>
-2. Copy the `redistrib_*.json` corresponding to the release to `pkgs/development/compilers/cudatoolkit/redist/manifests`.
-3. Generate the `redistrib_features_*.json` file by running:
+2. Make a note of the new version of CUDA available.
+3. Run
 
-    ```bash
-    nix run github:ConnorBaker/cuda-redist-find-features -- <path to manifest>
-    ```
+   ```bash
+   nix run github:connorbaker/cuda-redist-find-features -- \
```

Comment:
Not a blocker for this PR, but it would be nice for this to evenetually reside under NixOS or nix-community organization. In case a person becomes unavailable for a period of time, we have ways to still correct issues in the long term.

Maybe a follow-up task post PR @Mic92 @zimbatm @NixOS/cuda-maintainers  for thoughts about long-term cuda supoport within Nixpkgs (I know nvidia isn't the most OSS friendly, but there's a lot of compelling ML and enterprise use cases for CUDA).

Reply:
Yes, that was the idea. Originally, @ConnorBaker had implemented the tool in-tree in nixpkgs, but then it seemed like a good idea to move it out for faster/more flexible development

No reason not to move it to nix-community already is there, @ConnorBaker?

Reply:
None at all! Let me know what's involved in making that happen :)

I'd also appreciate feedback on it, especially given that I plan to do something similar to help out on the Apple SDK side of Nixpkgs. (I have #229210 which hasn't gotten much of my attention recently.)

Reply:
Added an item under future work to track the move